prop.properties

```diff
@@ -0,0 +1,8 @@
+url = https://chpl.ahrqdev.org/
```

Comment:
We can't set the URL permanently in a property file because we sometimes want to run tests against DEV, sometimes against STG, sometimes against PROD, and sometimes against local environments. That's why that isn't a property